### PR TITLE
JTao photospp in GeneratorInterface for 53X

### PR DESCRIPTION
**Pull request by JunquanTao**
Modified ExternalDecays/src/ExternalDecayDriver.cc
   Added PhotosInterface/interface/PhotosppInterface.h
   Added PhotosInterface/plugins/PhotosppInterface.cc
   Added PhotosInterface/test/DYnorad_Tune4C_13TeV_pythia8.py and PhotosInterface/test/DYphotospp_Tune4C_13TeV_pythia8.py  for test examples

### CODE REVIEW

**Comment by perrozzi** [comment]
please test with cms-sw/cmsdist#3521

File: GeneratorInterface/PhotosInterface/test/DYnorad_Tune4C_13TeV_pythia8.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    comEnergy = cms.double(13000.0),
+    crossSection = cms.untracked.double(0.2188),
+    maxEventsToPrint = cms.untracked.int32(1),
+    PythiaParameters = cms.PSet(
+        processParameters = cms.vstring('Main:timesAllowErrors = 10000', 
+					'ParticleDecays:limitTau0 = on', 
+					'ParticleDecays:tauMax = 10', 
+					'Tune:ee 3', 
+					'Tune:pp 5',
+					'PartonLevel:FSR = off',
```
Comment (perrozzi) [inline GeneratorInterface/PhotosInterface/test/DYnorad_Tune4C_13TeV_pythia8.py]: I don't  recall this being the only flag to set off for norad, can I ask what is the specific purpose of this cfg?

Comment (alberto-sanchez) [reply GeneratorInterface/PhotosInterface/test/DYnorad_Tune4C_13TeV_pythia8.py]: @perrozzi the idea is have two cfgs, one with photos++ enabled and one without it in order to compare the output and see the extra photons in there due to the photos++.

Comment (perrozzi) [reply GeneratorInterface/PhotosInterface/test/DYnorad_Tune4C_13TeV_pythia8.py]: ok, thanks (I had not noticed the same flag in the other cfg)

**Comment by perrozzi** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**